code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -135,8 +135,16 @@
 /obj/item/reagent_containers/borghypo/basic
 	name = "Basic Medical Hypospray"
 	desc = "A very basic medical hypospray, capable of providing simple medical treatment in emergencies."
-	reagent_ids = list("salglu_solution", "epinephrine")
+	reagent_ids = list("salglu_solution", "epinephrine", "sal_acid", "charcoal")
 	total_reagents = 30
 	maximum_reagents = 30
 
+/obj/item/reagent_containers/borghypo/basic/upgraded
+	name = "Upgraded Medical Hypospray"
+	desc = "Upgraded medical hypospray, capable of providing standart medical treatment."
+	reagent_ids = list("salglu_solution", "epinephrine", "spaceacillin", "sal_acid",
+	"charcoal", "hydrocodone", "mannitol", "salbutamol", "styptic_powder")
```

Comment:
Although the borg hypospray currently doesn't cause any reactions when injecting these chemicals, that's most likely an oversight - and if fixed, the Styptic Powder here would become a poison

Medical borgs already have trauma kits, so I'm not sure it's even necessary to add?

Reply:
Are you proposing to temporarily take away the hypospray from the Medborgs? Then they will ask chemists to cook the chemicals. Sounds good too.
This is for the future anyway.